app/home/page.tsx

```diff
@@ -29,7 +29,7 @@ const Home = () => {
     };
 
     return (
-        <div className="content">
```

Comment:
Keep using "content" className in every page to ensure the same layout for every page.